dll/win32/msgina/CMakeLists.txt

```diff
@@ -22,6 +22,6 @@ add_library(msgina MODULE
 set_module_type(msgina win32dll UNICODE)
 target_link_libraries(msgina wine uuid ${PSEH_LIB} cpprt atl_classes)
 add_delay_importlibs(msgina secur32)
-add_importlibs(msgina advapi32 user32 gdi32 powrprof userenv msvcrt kernel32 ntdll)
+add_importlibs(msgina advapi32 user32 gdi32 powrprof userenv msvcrt kernel32 ntdll uxtheme)
```

Comment:
It seems `uxtheme` should be added earlier, not after `ntdll`.

Reply:
should be moved just after userenv.

Reply:
Done.

Reply:
Also, since msgina is a core system dll, it's generally not recommended to add such user/theme-oriented dll import libs like that, but to load the dll manually at runtime and fall back when it doesn't exist.

Reply:
> Also, since msgina is a core system dll, it's generally not recommended to add such user/theme-oriented dll import libs like that, but to load the dll manually at runtime and fall back when it doesn't exist.

OK.